dateparser_data/supplementary_language_data/date_translation_data/ru.yaml

```diff
@@ -56,7 +59,34 @@ relative-type:
         - послезавтра
 
 simplifications:
-    - ^час: 1 час
-    - ^минуту: 1 минуту
-    - ^секунду: 1 секунду
+    - од(на|ну|ни|ной|ин): '1'
+    - дв(а|е|ое|ух): '2'
+    - пар[ауы]: '2'
+    - три: '3'
+    - четыре: '4'
+    - пять: '5'
+    - шесть: '6'
+    - семь: '7'
+    - восемь: '8'
+    - девять: '9'
+    - десять: '10'
+    - одиннадцать: '11'
+    - двенадцать: '12'
+    - пятнадцать: '15'
+    - двадцать: '20'
+    - тридцать: '30'
+    - сорок: '40'
+    - пятьдесят: '50'
     - несколько секунд: 44 секунды
+    - полчаса: 30 минут
+    - полгода: 6 месяцев
+    - полтора часа: 90 минут
+    - полтора года: 18 месяцев
+    - ((?<=(через|спустя|в течение)\s+)секунд[уы]|(?<=[^\d]\s+|^)секунду(?=(\s+назад))): 1 секунду
+    - ((?<=(через|спустя|в течение)\s+)минут[уы]|(?<=[^\d]\s+|^)минуту(?=(\s+назад))): 1 минуту
+    - ((?<=(через|спустя|в течение)\s+)часа?|(?<=[^\d]\s+|^)час(?=(\s+назад))): 1 час
+    - ((?<=(через|спустя|в течение)\s+)(день|дня)|(?<=[^\d]\s+|^)день(?=(\s+назад))): 1 день
+    - ((?<=(через|спустя|в течение)\s+)сут(ки|ок)|(?<=[^\d]\s+|^)сутки(?=(\s+назад))): 1 сутки
+    - ((?<=(через|спустя|в течение)\s+)недел[юи]|(?<=[^\d]\s+|^)неделю(?=(\s+назад))): 1 неделю
+    - ((?<=(через|спустя|в течение)\s+)месяца?|(?<=[^\d]\s+|^)месяц(?=(\s+назад))): 1 месяц
+    - ((?<=(через|спустя|в течение)\s+)года?|(?<=[^\d]\s+|^)год(?=(\s+назад))): 1 год
```

Comment:
The match is empty for ``regex.match('((?<=(через|спустя|в течение)\s+)года?|(?<=[^\d]\s+|^)год(?=(\s+назад)))', 'через год')``, is this expected or I'm testing it incorrectly?

Reply:
`regex.match` tries to apply the pattern at the beginning of the string. Since this pattern starts with a positive lookbehind, `regex.match` returns `None`. In this case `regex.search` is a more appropriate choice to check for a match because it's searching through the string.

Reply:
Aha I see, indeed, thanks for the explanation 👍